package.json

```diff
@@ -80,23 +83,23 @@
     "glotpress": "gulp glotpress",
     "changelog": "./vendor/bin/changelogger add"
   },
-  "engines": { "node": "18.13.0", "npm": "8.19.3" },
+  "engines": {
+    "node": "18.17.0",
+    "npm": "9.6.7"
+  },
   "devDependencies": {
     "@playwright/test": "^1.44.0",
-    "@the-events-calendar/product-taskmaster": "^3.0.0",
+    "@redux-devtools/extension": "^3.3.0",
     "@types/node": "^20.10.6",
     "@wordpress/eslint-plugin": "^17.2.0",
-    "@wordpress/hooks": "^1.3.2",
+    "@wordpress/hooks": "*",
     "cross-env": "^5.2.0",
     "dotenv": "^16.3.1",
-    "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.6.0",
     "gettext-parser": "1.3.0",
     "husky": "^1.3.1",
```

Comment:
Can remove - never caught on.
There's probably a bit more to do alongside that though

Reply:
In these PRs I'm going for the bare minimum to get the code building and running correctly. I will address more corpose changes in the PR to move the build system to `@wordpress/scripts`.